### DISCUSSION TITLE
Alpha 164

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4941] [Release Notes]
- fixed window buttons alignment (Kozik Przemysław) [#4928](https://github.com/Eugeny/tabby/pull/4928)
- macOS: handle `dscl` errors
- added support for long-clicking for context menu when right click is set to paste 
- config sync is experimental no more

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.164'>Alpha 164</a>.</em>

**Comment by pinpins** [comment]
Hi,

 small observation with latest release, before in order to paste I could right click anywhere on a terminal, now I should do it in a empty space, as otherwise right click selects text in terminal now..

As if that is part of new feature, would appreciate some toggle setting to turn it off

Thanks, pinpin

**Reply by Eugeny** [reply]
Hi, are you using macOS?

**Reply by pinpins** [reply]
That's correct

**Reply by Eugeny** [reply]
Automatic word selection has always been there and isn't new to v164 - it's the same as in other macOS apps. You can use Cmd-V to reliably paste at any time.

**Reply by pinpins** [reply]
It was not selecting for right-click before, just pasting directly. Left-click yes, it was there always

**Comment by xingyo** [comment]
no support linuxmint
